tests/testthat/_snaps/assert.md

```diff
@@ -179,14 +179,14 @@
     Output
       <error/vctrs_error_cast_lossy>
       Error in `vec_assert()`:
-      ! Can't convert from `size` <double> to <integer> due to loss of precision.
+      ! Can't convert from `size` <double> to `integer()` <integer> due to loss of precision.
```

Comment:
I don't love this one. I think it was much nicer when we just had `to = ""` which resulted in no arg being displayed here

Reply:
See next comment about updating the lossy cast message

Reply:
These are just unit tests, they don't reflect how `vec_cast()` is supposed to be used.